### ISSUE TITLE
Config overwritten after 3.0 update

### ISSUE THREAD

**Issue by datosh** [zeit/hyper#3612]
- [ ] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [ ] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Windows 10 1809
- **Hyper.app version**: 3.0.0 stable
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: default config
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: Vanilla

## Issue
Hyper automatically updated to 3.0.0.
When I opened Hyper this morning I got the Vanilla hyper.js without any config. After checking the `%APPDATA/Hyper` folder I found that my config was overwritten with the default.

As far as I understand the release notes this should have been fixed by canary10
https://github.com/zeit/hyper/releases/tag/3.0.0-canary.10

**Comment by williamoverton** [comment]
Yeah not good :/ I didnt have a backup of my config. Also all my installed plugins are no longer active.

**Comment by williamoverton** [comment]
Update! If I run hyper from the command line (by just typing hyper) it has all my old config and plugins still!

**Comment by ButchMonkey** [comment]
Looks like the .hyper.js file was re-created in `appdata/roaming/hyper/.hyper.js` where as before it was installed under the user.

By running from the command line, you will run the old version (I assume the path variable hasnt been updated)

**Comment by snassour** [comment]
I'm having this issue as well, but launching hyper from the command line didn't change anything. How can I fix it?

**Comment by pcnate** [comment]
@snassour try opening `%localappdata%\hyper\` and seeing if you can launch the old version from explorer

**Comment by voronoipotato** [comment]
likely duplicate https://github.com/zeit/hyper/issues/3589
